homeassistant/components/dsmr_reader/sensor.py

```diff
@@ -19,7 +23,26 @@ async def async_setup_platform(
     async_add_entities: AddEntitiesCallback,
     discovery_info: DiscoveryInfoType | None = None,
 ) -> None:
-    """Set up DSMR Reader sensors."""
+    """Set up DSMR Reader sensors via configuration.yaml and show deprecation warning."""
+    _LOGGER.warning(
```

Comment:
Change to use repair instead.

Reply:
It wasn't available when I originally submitted the pull request 😅 But I'll look into implementing it like that, as it would indeed be a lot cleaner. I was planning on doing that in a separate pull request, just to finally get this one merged. But I guess it shouldn't be too much work.

Reply:
Should be pretty straight forward and there is plenty of examples to just copy.

Reply:
Done. I just wasn't sure of the value to set in `breaks_in_ha_version`. I assume 2 months (hoping it gets included in the 2022.10 release) would be sufficient for such a minor configuration change.

Reply:
That's normally how it goes that you set the next version and in case it wouldn't make the cut it's an easy adjustment to the next one. But this looks like good to go now.